src/bitcoin-cli.cpp

```diff
@@ -422,6 +422,10 @@ static int CommandLineRPC(int argc, char *argv[])
             gArgs.ForceSetArg("-rpcpassword", rpcPass);
         }
         std::vector<std::string> args = std::vector<std::string>(&argv[1], &argv[argc]);
+#ifdef WIN32
+        for (std::string& arg : args)
+            arg = AnsiToUtf8(arg);
+#endif
```

Comment:
In order to solve this problem, there are three ways to solve this.
- The first is above, which might get `?` if the user enter any character that can't present by user's code page.
- The second is Use `GetCommandLineW` and `CommandLineToArgvW` to get command line options by `wchar_t`, which can present more Unicode characters.
- The third one is add wmain to get command line in `wchar_t` directly by "(w)main" function.

Which one is prefered?

Reply:
I choose the third way